db/primary_migrate/20220902162411_add_allow_threatmetrix_to_service_providers.rb

```diff
@@ -0,0 +1,10 @@
+class AddAllowThreatmetrixToServiceProviders < ActiveRecord::Migration[7.0]
+  def up
+    add_column :service_providers, :allow_threatmetrix, :boolean
```

Comment:
Cc @18F/identity-ursula does this property name match the style used for other per-SP configs? Would `device_profiling_enabled` be clearer?

Reply:
Modeled it after `allow_prompt_login` but there is a `irs_attempts_api_enabled` so i'm up for either

Reply:
`allow_prompt_login` lets client apps select a new behavior based on request params, so it is an "allow". This is something enabled on the backend, no per-request limit

Reply:
Makes sense.  Are you sure you want it `device_profiling_enabled` and not `threatmetrix_enabled`?

Reply:
I actually added that `allow_prompt_login` :-)

Reply:
What's the verdict? We now have 3 very similar names that are sometimes used interchangeably: threatmetrix, device_profiling, and ddp.

Reply:
Naming is so hard. I personally think that `device_profiling_enabled` is the clearest and easiest to read. Is there concern that this flag might control more than just device profiling in the future?

Reply:
I agree with @NatashaJIbrahim - since it's not something the partner needs to do anything with other than asking us to turn it on, `enabled` feels better than `allow`. To your question, @NatashaJIbrahim, I would suggest that if we need to control something else on a per-SP level in the future, we can add it as a separate option 😄.